### ISSUE TITLE
Random initialisation learning rate

### ISSUE THREAD

**Issue by Stephanie-Shen324** [valeoai/SLidR#11]
Hi, would you mind providing the lr for random initialisation? for both kitti & nuscenes dataset. Thank you!

**Comment by CSautier** [comment]
On NuScenes we used 0.05 for all experiments with random initialisation. I believe we also used 0.05 on SemanticKITTI, but I will have to check with a co-author currently in vacation.
Don't forget to set lr_head to Null as this setting does not make sense without pre-training.

**Comment by Stephanie-Shen324** [comment]
<img width="448" alt="github" src="https://user-images.githubusercontent.com/72006612/185898143-b725c5ad-b5f9-4bf3-b425-cb5af9d4fce3.png">

1. The scenario that I am asking is to reproduce random initialisation results as above in red circle. I believe we should look for larger learning rates like 3 or 4. (For example, I have tried both lr set to 3.5 on KITTI. It gave me mIOU=0.3188 and I am expecting higher mIOU with higher learning rate.) Your answer seems to be the config for linear probing?  

2. For linear probing, shouldn't it be lr=Null & lr_head=0.05? as in linear probing we want to only fine-tune the segmentation head and freeze the other layers.

**Comment by CSautier** [comment]
The 1% result on NuScenes was obtained with lr=0.05. It is possible that increasing the learning rate could increase the results a bit in very low data regime, however 0.05 was found to be optimal or close to optimal at nearly all proportion on NuScenes with random initialisation. I would think that instead of increasing the learning rate, increasing the number of epochs would make more sense.

As for linear probing parameters, if you specify lr_head=Null, then the same learning rate will be applied for all weights. I don't think you can put lr=Null as pytorch_lightning wouldn't accept it. However you have to specify freeze_layers : True, as there are also the batchnorms that have to been taken care of, and when you do that, it freezes the training for everything but the head.

**Comment by Stephanie-Shen324** [comment]
Thank your for the clarifications!
